lib/fog/aws/parsers/compute/describe_images.rb

```diff
@@ -66,9 +66,9 @@ def end_element(name)
               end
             else
               case name
-              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType'
+              when 'architecture', 'description', 'hypervisor', 'imageId', 'imageLocation', 'imageOwnerAlias', 'imageOwnerId', 'imageState', 'imageType', 'kernelId', 'name', 'platform', 'ramdiskId', 'rootDeviceType','rootDeviceName','virtualizationType','creationDate'
```

Comment:
The SDK returns name this way "2014-12-01T19:31:49.000Z". Can I convert the time format to '%FT%TZ' is that fine for time format, or  is there another standard format I need to use?